### ISSUE TITLE
request: add vuescan-bin

### ISSUE THREAD

**Issue by ryanmusante** [chaotic-aur/packages#122]
aur: https://aur.archlinux.org/packages/vuescan-bin/

**Comment by PedroHLC** [comment]
I've added this package, but it'll have to keep this issue open until I push it to this repo...

**Comment by ryanmusante** [comment]
Does it get tested before being added to repo?

**Comment by PedroHLC** [comment]
Nope, that's with you (or AUR maintainer adding check function) :smile:
But this is a -bin package, and we can't do much as it's pretty much a "recompression".
Is it working?

**Comment by ryanmusante** [comment]
Last time I tried it failed, I need to talk to the maintainer. I apologize, please put this on hold.
